app/policies/admin/reader_policy.rb

```diff
@@ -3,7 +3,8 @@
 module Admin
   # Policy for managing readers
   class ReaderPolicy < ApplicationPolicy
-    # manage? is the default rule.
```

Comment:
This was misleading, suggesting that all rules would somehow get the permissions granted by `manage?`.

Reply:
Yeah, it is really confusing that default rules don't cover everything. That trips me up all the time with AP.